Client/Frontend/Metadata Parsing/MetadataParserHelper.swift

```diff
@@ -51,5 +101,17 @@ class MetadataParserHelper: TabContentScript {
 
         TabEvent.post(.didLoadPageMetadata(pageMetadata), for: tab)
         NotificationCenter.default.post(name: NotificationOnPageMetadataFetched, object: nil, userInfo: userInfo)
```

Comment:
Using `NotificationOnPageMetadataFetched` should probably be deprecated in favour of `TabEvent`s.